File: Mobile/SocionicTeamBuilder.Mobile/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SocionicTeamBuilder.Mobile.Models
{
    public class Employee
    {
        public Employee() { }

        public int Id { get; set; }
        public string Login { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string FullName { get; set; }
        public string SocionicType { get; set; }
        public DateTime DateOfBirth { get; set; }

        public string Enterprise { get; set; }
    }
}

File: Frontend/src/app/team-creator-panel/employee/employee.ts
export class Employee {
    constructor(
        public id: number,
        public enterprise: string,
        public login: string,
        public password: string,
        public email: string,
        public role: string,
        public fullName: string,
        public socionicType: string,
        public workingProfile: string
    ) {}
}